server/router.js

```diff
@@ -95,5 +96,85 @@ router.get('/', function(req, res) {
   }
 });
 
+let defined = function(variable) {
+  if (typeof variable !== 'undefined' && variable !== null) {
+    return true;
+  }
+  return false;
+};
+
+router.post('/', function(req, res) {
+  var body = '';
+
+  req.on('data', function (data) {
+    body += data;
+    // Too much POST data, kill the connection!
+    // 1e6 === 1 * Math.pow(10, 6) === 1 * 1000000 ~~~ 1MB
+    if (body.length > 1e6) {
+      req.connection.destroy();
+    }
+  });
+
+  req.on('end', function () {
+    var post = qs.parse(body);
+
+    if (defined(post['url']) === false) {
+      res.status(400);
+      res.send('you shall not pass.');
+    } else {
+      let apiUrl = post['url'];
```

Comment:
Rather than passing the `url` through the POST body, can we just use a query parameter so that the proxy usage is consistent whether it's a GET or a POST?